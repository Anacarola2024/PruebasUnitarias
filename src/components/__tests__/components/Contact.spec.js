import { mount } from '@vue/test-utils';
import { describe, it, expect } from 'vitest';
import ContactView from '@/views/ContactView.vue';

describe('ContactView.vue', () => {
  it('matches the snapshot', () => {
    const wrapper = mount(ContactView, {
      props: {
        title: 'Contact Us',
        description: 'This is the contact page.'
      }
    });

    // Comparar el HTML del componente con el snapshot guardado
    expect(wrapper.html()).toMatchSnapshot();
  });
});