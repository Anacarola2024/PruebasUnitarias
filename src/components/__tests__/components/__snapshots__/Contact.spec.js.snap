// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Contact.vue > matches the snapshot 1`] = `"<h1>This is a Contact page</h1>"`;

exports[`ContactView.vue > matches the snapshot 1`] = `"<h1 title="Contact Us" description="This is the contact page.">This is a Contact page</h1>"`;
